app/bundles/EmailBundle/Tests/EventListener/EmailSubscriberTest.php

```diff
@@ -101,4 +102,28 @@ public function testOnEmailResendWhenShouldNotTryAgain(): void
         $this->subscriber->onEmailResend($queueEmailEvent);
         $this->assertFalse($queueEmailEvent->shouldTryAgain());
     }
+
+    public function testOnEmailFailed(): void
+    {
+        $this->mockSwiftMessage->leadIdHash = 'idhash';
```

Comment:
Can we do something like this:
```
        $message               = new class() extends \Swift_Message {
            public $leadIdHash = 'some-hash';
        };
```

Instead of modifying the actual class to make a test work?